src/Exception/OutOfBoundsException.php

```diff
@@ -0,0 +1,12 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\AreabrickConfigBundle\Exception;
+
+class OutOfBoundsException extends \Exception
```

Comment:
Please use the PHP core exception instead of defining a custom one: `\OutOfBoundsException`